src/Actions/RelativeLineNumbers.ts

```diff
@@ -11,6 +11,12 @@ export class ActionRelativeLineNumbers {
                     return Promise.resolve(false);
                 }
 
+                const lineNumbersTurnedOff = activeTextEditor.options.lineNumbers === TextEditorLineNumbersStyle.Off
+
+                if (lineNumbersTurnedOff) {
+                    return Promise.resolve(false)
```

Comment:
We should use `resolve(false)` here because we are created a `Promise` outside.
Line 11 is also wrong (My fault).

Reply:
Ah good spot!